win32ss/user/ntuser/msgqueue.c

```diff
@@ -1771,6 +1771,8 @@ BOOL co_IntProcessKeyboardMessage(MSG* Msg, BOOL* RemoveMessages)
     BOOL Ret = TRUE;
     WPARAM wParam = Msg->wParam;
     PTHREADINFO pti = PsGetCurrentThreadWin32Thread();
+    UINT language_hotkey;
+    BOOL change_language = FALSE;
```

Comment:
code style is: `LanguageHotkey` and `ChangeLanguage` (or `bChangeLanguage`)

Reply:
Done